bench.rb

```diff
@@ -50,6 +50,13 @@
     end
   RUBY
 
+  x.report "solid_queue-pushbulk", <<~RUBY
+    ActiveJob::Base.queue_adapter = :solid_queue
+    ActiveJob.perform_all_later jobs.times.map do
+      RoundupJob.new(123, "hello world", hash)
+    end
```

Comment:
```suggestion
    ActiveJob.perform_all_later jobs.times.map {
      RoundupJob.new(123, "hello world", hash)
    }
```

or 

```suggestion
    ActiveJob.perform_all_later(jobs.times.map do
      RoundupJob.new(123, "hello world", hash)
    end)
```

Otherwise you are calling
```suggestion
    ActiveJob.perform_all_later(jobs.times.map) do
      RoundupJob.new(123, "hello world", hash)
    end
```
Which just fails.